API-Samples/android/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2016 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmake_minimum_required(VERSION 3.4.1)
+get_filename_component(SHADERC_SRC
+                       ${ANDROID_NDK}/sources/third_party/shaderc
+                       ABSOLUTE)
+# Build Util Lib
+set(UTILS_NAME  vsamputils)
+set(GLSLANG_SPIRV_INCLUDE_DIR ${SHADERC_SRC}/third_party/glslang)
+get_filename_component(PROJECT_SOURCE_DIR
+                       "${CMAKE_SOURCE_DIR}/../../.."
+                       ABSOLUTE)
+get_filename_component(GLMINC_PREFIX
+                       "${CMAKE_SOURCE_DIR}/../utils"
+                       ABSOLUTE)
+add_subdirectory(${CMAKE_SOURCE_DIR}/../utils ${CMAKE_BINARY_DIR}/utils)
```

Comment:
Thanks.  I felt that the gradlew is typed with this version of gradlew we are using ( 2.14.1 ): safer this way since we do have a build system change underneath ( experimental --> cmake ).